event_sale_pro/views/event_registration.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<record model="ir.ui.view" id="view_event_registration_form">
+    <field name="name">Source document links</field>
+    <field name="model">event.registration</field>
+    <field name="inherit_id" ref="event.view_event_registration_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//sheet/group">
```

Comment:
You want to use the implicit position `inside`? It would be clearer to have it explicit.